scripts/bert/finetune_classifier.py

```diff
@@ -148,11 +148,22 @@
 
 # data type with mixed precision training
 if args.dtype == 'float16':
+<<<<<<< HEAD
     from mxnet.contrib import amp  # pylint: disable=ungrouped-imports
     # monkey patch amp list since topk does not support fp16
     amp.lists.symbol.FP32_FUNCS.append('topk')
     amp.lists.symbol.FP16_FP32_FUNCS.remove('topk')
     amp.init()
+=======
+    try:
+        # monkey patch amp list since topk does not support fp16
+        amp.lists.symbol.FP32_FUNCS.append('topk')
+        amp.lists.symbol.FP16_FP32_FUNCS.remove('topk')
+        amp.init()
+    except ValueError:
+        # topk is already in the FP32_FUNCS list
+        amp.init()
+>>>>>>> 5e11334f5c00fd2875ab75c670b2595c560c30fc
```

Comment:
@zburning during the `git merge`, you can edit the code to resolve the conflict. Now you commited the unresolved / conflicting files. You can see https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging about the normal process. Now you may simply add a new commit to remove the conflicting code.